### PR TITLE
setup.py: Add PyPI trove classifiers for supported Python versions

### PR DESCRIPTION
**Pull request by cclauss**
This helps users of PyPI, pyup, etc. to know what versions of Python are supported.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,13 @@
 Intended Audience :: Information Technology
 License :: OSI Approved :: BSD License
 Programming Language :: Python
+Programming Language :: Python :: 2
```
Comment (Wooble) [inline setup.py]: We also test on Travis on 2.6 and 3.3.

**Comment by Wooble** [comment]
We might also consider setuptools python_requires argument to setup(), although I'm not 100% confident it's in setuptools versions that work with all of the versions our codebase will technically run on.

(This sort of leads to the question of whether we should explicitly drop support for old versions of python.)

File: setup.py (hunk 1 above)
Comment (cclauss) [reply setup.py]: Really??  2.6 EOL was five years ago and 3.3 was more than a year ago.

https://snarky.ca/stop-using-python-2-6 was written three years ago.

Comment (Wooble) [reply setup.py]: Technically there's code that's only necessary in 2.4 and before, although I'm not even confident the rest of the codebase will work on 2.4 at all, and I'm certainly not going to propose we adjust our CI to enforce that it does.  It's just that no one's gotten around to removing it, and it doesn't particularly hurt anything as it is. Probably.

I personally wouldn't object to dropping support for older versions and Mr Cannon as usual makes some good points.

**Pull request merged**